api/organization.js

```diff
@@ -13,10 +13,31 @@ const storage = multer.diskStorage({
 })
 const upload = multer({storage: storage});
 const { headerConstants } = require("../config/constants/header_constants.js");
-const {createNewOrg,updateOrg} = require("../services/organization")
+const {createNewOrg, updateOrg, getOrganization} = require("../services/organization")
 
 const API_KEY = process.env.API_KEY;
 
+router.get("organization", async (req,res)=>{
+    try {
+        //check token
+        const token = req.get(headerConstants.tokenHeader);
+        if (token === null|| token === undefined || token === "" || typeof token !== "string") {
+            return res.sendStatus()
+        }
+
+        //business logic
+        const result = await getOrganization(token);
+        res.header = "200";
+        res.send(result);
```

Comment:
Hmm are you sure about this, the style is suppose to be consistent

Reply:
Since res.send() will send an 200 status code so i think we dont need the line "res.header = 200" right ?

Reply:
I think I should be more specific but it's okay LGTM